tests/framework/framework.go

```diff
@@ -139,6 +148,13 @@ func (f *Framework) BeforeEach() {
 		f.CdiClient = cs
 	}
 
+	if f.ControllerPod == nil {
+		pod, err := utils.FindPodByPrefix(f.K8sClient, f.CdiInstallNs, CdiPodPrefix, "")
+		Expect(err).NotTo(HaveOccurred())
+		fmt.Fprintf(GinkgoWriter, "INFO: Located cdi-controller-pod: %q\n", pod.Name)
+		f.ControllerPod = pod
+	}
```

Comment:
I wonder if it's better to look up the controller only when you need it.  What if we want to write tests that redeploy all/part of CDI?

Reply:
Then we call FindPodByPrefix for that particular scenario. On the other hand I can think of every single group of tests a use for having the controller pod available (mostly to get the log)

Reply:
@awels is the goal here to have the framework ensure that the cdi controller is running (or at least present), and to do that only once per test binary?

Reply:
@aglitke maybe this should be another `Config{}` field: whether to lookup the import controller pod?

Reply:
No the goal is to get the pod name of the controller, so we can inspect it for certain tests (like improve the did nothing happen test by inspecting the log and noting the controller saw and rejected the PVC).

Reply:
@jeffvance yeah that would be a good enhancement.